charts/datafold/charts/storage-worker/values.yaml

```
@@ -17,6 +17,7 @@ worker:
   tasks: 200
```

```suggestion
  tasks: 1
```

If I understand the meaning of this setting?

---

@anatoly-scherbakov : Works. I'll update.